### PR TITLE
Set modules path to load without installing

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

**Comment by patrickelectric** [comment]
Up.

**Comment by peterbarker** [comment]
Does this work with Python3?

**Comment by patrickelectric** [comment]
I did this changes in my arch linux machine, that runs by default python3.

**Comment by patrickelectric** [comment]
@peterbarker ^

**Comment by patrickelectric** [comment]
ping @peterbarker

File: MAVProxy/mavproxy.py [hunk 1]
```diff
@@ -464,6 +464,11 @@ def import_package(name):
     """Given a package name like 'foo.bar.quux', imports the package
     and returns the desired module."""
     import zipimport
+
+    # Set the modules path to search for modules
+    src_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'modules')
```
Comment (peterbarker) [inline MAVProxy/mavproxy.py]: If MAVProxy is installed - aren't we essentially adding a random path to the search path here?

**Pull request closed without merging**